rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book
```

Comment:
- KIndly, Add Getters and Setters for @person instance variables should for later initialization :+1: